model/Core/Vocabularies/RelationshipType.md

```diff
@@ -25,10 +25,12 @@ between a Package and a File, between two Packages, or between one SPDXDocument
 - buildDependency: Every `to` Element is a build dependency of the `from` Element
 - buildTool: Build tool used to build an Element. This may be used to describe the build tool of a Build instance
 - coordinatedBy: (Security) Used to identify the vendor, researcher, or consumer agent performing coordination for a vulnerability
+- concludedLicense: Identifies the license that that SPDX data creator has concluded as governing the software Package, File or Snippet.
```

Comment:
... "governing the software Artifact."

Reply:
Please be explicit, what is "to" and what is "from" here